env/common/templates/galaxy/config/job_conf.yml.j2

```diff
@@ -65,10 +65,10 @@ execution:
       function: map_tool_to_destination
       rules_module: tpv.rules
       tpv_config_files:
+        - "https://gxy.io/tpv/db.yml"      
```

Comment:
Can we version this as anything else ? Otherwise I'd prefer to just use a git commit. We don't want to update these as we update Galaxy, which has the potential to go wrong. I'm a big fan of one change at a time.

Reply:
Let's pin it to a commit for now. That way, you can control updates at your pace. In the meantime, we can see what kind of versioning scheme would make sense in the TPV shared db repo.

My own view is that a constant url that people can refer to and always get the latest version has a lot of value. There's just less maintenance that way. TPV relies on testing to ensure that behaviour is consistent. The original idea was that if we introduce any breaking change, the tests will fail, and that's the point at which a new version should be minted. I think what you're looking for is an even greater level of preciseness and control. For now, that can be achieved through commit pinning. In future, we can figure out a good automated release process.

Reply:
> My own view is that a constant url that people can refer to and always get the latest version has a lot of value.

Not saying it's not, but there's a reason one typically doesn't deploy an application from dev. This is fine for dev instances, but not for production.

> The original idea was that if we introduce any breaking change, the tests will fail

I like that as an aspirational goal, but tpv-lint doesn't cover enough ground to be confident at this point.